project/stac_api/models.py

```diff
@@ -27,17 +26,19 @@
 # instance so that it's not shared between all field instances.
 # HINT: Use a callable instead, e.g., use `list` instead of `[]`.""
 # <end quote>
+DEFAULT_STAC_EXTENSIONS = { # pylint: disable=invalid-name
```

Comment:
The pylint pragma is not anymore useful here :-)

Reply:
removed :-)